File: src/Models/Credential.cs
namespace CineComplex.Models
{
    public class Credential
    {
        private string? _loginId;
        private string? _password;
        //private string? _type;
        private string? _sessionTokenId;
        private static Credential? _instance;

        


        private Credential()
        {
        }

        public string? LoginId { get => _loginId; set => _loginId = value; }
        public string? Password { get => _password; set => _password = value; }
        public string? SessionTokenId { get => _sessionTokenId; set => _sessionTokenId = value; }
        public static Credential Instance
        {
            get
            {
                if (_instance == null)
                {
                    _instance = new Credential();
                }
                return _instance;

            }
        }

    }
}

File: src/Models/Credential.py
# from sqlalchemy import Column, Integer, String
# from Database.models import Base
#
# from src.Classes.Base.ABaseModel import ABaseModel
#
#
# class Credential(ABaseModel):
#     __tablename__ = "credentials"
#
#     id = Column(Integer, primary_key=True, index=True)
#     login_id = Column(String, unique=True, nullable=False)
#     password_hash = Column(String, nullable=False)
#     session_token_id = Column(String, unique=True, nullable=True)
#
#     def __repr__(self):
#         return f"Credential(ID={self.id}, LoginID={self.login_id}, Token={self.session_token_id})"
